package.json

```diff
@@ -33,6 +33,7 @@
     "vite": "^5.2.0"
   },
   "dependencies": {
+    "axios": "^1.7.2",
```

Comment:
If we re-use the existing product service from v1 we won't need to use axios. That code was just using `fetch`